Hyperspace.xcodeproj/project.pbxproj

```diff
@@ -644,6 +664,13 @@
 		60D6C3E6201CD77400B3B012 /* Frameworks */ = {
 			isa = PBXGroup;
 			children = (
+				CB3A954F21E5523000A70243 /* BrightFutures.framework */,
+				CB3A954D21E5522600A70243 /* BrightFutures.framework */,
+				CB3A954B21E5521B00A70243 /* BrightFutures.framework */,
+				CB0961E621E51894006CC9F5 /* FutureKit.framework */,
```

Comment:
Looks like there are some remnants of FutureKit in the project?

Reply:
Yeah, it's odd that it's there. It's not showing up in Xcode anywhere, but I'll edit the file manually.